related-files-buffer.el

```diff
@@ -0,0 +1,56 @@
+;;; related-files-buffer.el --- Provide methods and jumpers for buffers  -*- lexical-binding: t; -*-
```

Comment:
I don't think there are any "jumpers" provided in this file. Also, the notion of "place" could probably be added to the documentation. I would write something like:

"Add support for buffer-based places to related-files"

Reply:
Fixed in 7b56215